rcl/src/rcl/init.c

```diff
@@ -188,13 +188,13 @@ rcl_init(
       "Enclave name is not valid: '%s'. Invalid index: %zu",
       rcl_enclave_name_validation_result_string(validation_result),
       invalid_index);
-    fail_ret = RMW_RET_ERROR;
+    fail_ret = RCL_RET_ERROR;
```

Comment:
These changes should probably be in a separate PR. It's unrelated to test coverage.

Reply:
Sounds good. See https://github.com/ros2/rcl/pull/686.